### PR TITLE
Make error messages for using font.align clearer

### PR DESCRIPTION
**Pull request by Kn4ughty**
Hello, while using font.align in a project, I passed it bad data, and the error message was not clear, and I had to read the source code to work out what the error message actually meant. 
The problem is when you pass something that is not FONT_LEFT, FONT_CENTER, or FONT_RIGHT it says it must be an integer, but to the end user it might appear that FONT_LEFT is not an integer, and they might be confused about what they need to parse. 

I had a function that drew some text, and it was something like this
```python
self.fontObj.align = self.textJustify
```
Then when i passed it the wrong type, the error message said it must be an integer, and I thought "I thought it was pygame.FONT_LEFT!" 

I hope that this change makes it more clear to any users in the future.

### CODE REVIEW

**Comment by oddbookworm** [comment]
The CI fails I'm seeing so far are concerning (not to do with your code though)

**Review by oddbookworm — APPROVED** [review]
LGTM! The CI fails due to a missing dependency are weird and I've mentioned the fails to @ankith26 (the CI guru). Hopefully something can be done to fix those soon

File: src_c/font.c [hunk 1]
```diff
@@ -484,14 +484,17 @@ font_setter_align(PyObject *self, PyObject *value, void *closure)
...
         return -1;
     }
 
     if (val < 0 || val > 2) {
-        PyErr_SetString(
-            pgExc_SDLError,
-            "font.align should be FONT_LEFT, FONT_CENTER, or FONT_RIGHT");
+        PyErr_SetString(pgExc_SDLError,
+                        "font.align should be FONT_LEFT, FONT_CENTER, or "
+                        "FONT_RIGHT, (0, 1 or 2)");
```
Comment (Starbuck5) [inline src_c/font.c:497]: This PR is a good idea, but I'm hesitant to report the values of the constants in the error message.

The values of the constants should be an irrelevant detail.

**Comment by Starbuck5** [comment]
The error message is missing spaces in necessary locations:

```
>>> a.align = -1
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
pygame.error: font.align must be FONT_LEFT, FONT_CENTER, orFONT_RIGHT.
>>> a.align = "hello"
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: font.align must be an integer.Must correspond with FONT_LEFT, FONT_CENTER, or FONT_RIGHT
```

**Review by ankith26 — APPROVED** [review]
LGTM, thanks for contributing :sunglasses:

**Pull request merged**